app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsContributionsFragment.kt

```diff
@@ -179,28 +180,36 @@ class SuggestedEditsContributionsFragment : Fragment(), SuggestedEditsTypeItem.C
     }
 
     private fun getArticleContributionDetails() {
-        disposables.add(Observable.fromIterable(continuedArticlesContributions)
-                .subscribeOn(Schedulers.io())
-                .doFinally {
+        val summaryObservables = ArrayList<ObservableSource<PageSummary>?>()
+        for (contribution in continuedArticlesContributions) {
+            summaryObservables.add(ServiceFactory.getRest(contribution.wikiSite).getSummary(null, contribution.title).subscribeOn(Schedulers.io()).onErrorReturnItem(PageSummary()))
```

Comment:
Hint:  If you look in logcat, you will see that these `summaryObservables` are correctly working in parallel, but your `imageInfoObservables` below are working in series.  What's different about them?